docs/dev/dev_setup.rst

```diff
@@ -64,92 +77,97 @@ from your fork to your local computer where you'll be editing the code or docs.
 
-   If there have been changes to the environment on a branch (e.g. ``dev``) that you
-   merge into your own branch, the lockfiles will need to be regenerated. This can be
-   done automatically by pushing the merged changes to your branch on GitHub, waiting a
-   couple of minutes for the ``update-conda-lockfile`` GitHub Action to run, and then
-   pulling the fresh lockfiles to your local development environment. You can also
-   regenerate the lockfiles locally (see below).
+   **Never manually merge pixi.lock!** The lockfile (``pixi.lock``) is a
+   generated file that must be regenerated programmatically. The git configuration
+   above will automatically keep your version of the lockfile during merge conflicts,
+   but you must then regenerate it to reflect the merged dependencies.
+
+   After merging changes from another branch (e.g. ``main``) that modified
+   ``pyproject.toml``:
+
+   1. Complete the merge (the lockfile will keep your version due to the merge strategy)
+   2. Regenerate the lockfile to match the merged dependencies:
+
+   .. code-block:: console
+
+      $ pixi update
+
+   3. Commit the regenerated lockfile
+
+   This ensures your lockfile is consistent with the merged dependency specifications.
 
 -------------------------------------------------------------------------------
 Create the PUDL Dev Environment
 -------------------------------------------------------------------------------
-We use `conda-lock <https://github.com/conda/conda-lock>`__ to specify particular
-versions of all of PUDL's direct and indirect software dependencies in a lockfile,
-resulting in a stable, reproducible environment. This lockfile and several
-platform-specific rendered environment files are stored under the ``environments/``
-directory in the main PUDL repository.
-
-All of the dependencies in ``environments/conda-lock.yml`` are derived from packages
-listed in the project's ``pyproject.toml`` file.  The conda lockfile is updated
-automatically by a GitHub Action workflow that runs once a week, or any time
-``pyproject.toml`` is changed.
-
-We use a ``Makefile`` to remember and automate some common shared tasks in the
-PUDL repository, including creating and updating the ``pudl-dev`` conda environment. If
-you are on a Unix-based platform (Linux or MacOS) the ``make`` command should already be
-installed. You'll typically want to use the predefined ``make`` commands rather than
-running the individual commands they wrap. If you'd like to learn more about how
-Makefiles work, check out `this excellent Makefile tutorial
-<https://makefiletutorial.com/>`__
-
-To create the ``pudl-dev`` environment, install the local PUDL package, and activate the
-newly created environment, run:
+We use `pixi <https://pixi.sh>`__ to manage our development environment. Pixi uses a
+lockfile (``pixi.lock``) to specify particular versions of all of PUDL's direct and
+indirect software dependencies, resulting in a stable, reproducible environment.
```

Comment:
This is duplicative with the `Install pixi` section above - if we do cut the installation instructions maybe it makes sense to just merge the two sections?